### DISCUSSION TITLE
Group 7 Discussion Thread

### DISCUSSION THREAD

**Discussion by jgrunenberg** [Digital-Preservation-Coalition/PythonStudyGroups#25] [General]
February 27 meeting notes

1. learn git!
2. review checksums in python
3. learn practices for better managing repeatable actions so less copy and paste of blocks of re-usable code

**Comment by cnwalker89** [comment]
For March 26th meeting:

Bring a script you have been working on OR a use case you would like to create a script for.

**Comment by SholtoD** [comment]
Here is the link to the safemover script I mentioned today - https://github.com/NLNZDigitalPreservation/SafeMoverGUI

**Comment by lars-unimelb** [comment]
> February 27 meeting notes
> 
>     1. learn git!
> 
>     2. review checksums in python
> 
>     3. learn practices for better managing repeatable actions so less copy and paste of blocks of re-usable code

Btw, as I mentioned in the study group, you can import other python scripts, one example is from the fastapi documentation (https://fastapi.tiangolo.com/tutorial/sql-databases/)

Imagine you have 

```
└── sql_app
    ├── __init__.py
    ├── crud.py
    ├── database.py
    ├── main.py
    ├── models.py
    └── schemas.py
```

you can import models.py into main.py with 

`from . import models`

you can also use that to save different defs and call them in when needed.

**Comment by SholtoD** [comment]
https://docs.python.org/3/library/pathlib.html
https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation
https://discuss.python.org/t/error-if-full-path-to-file-more-than-256-symbols/33794
